### PR TITLE
SOC-? Modify db dump

### PR DESCRIPTION
**Pull request by slayful**

### CODE REVIEW

File: extensions/wikia/Discussions/maintenance/ForumDumper.php [hunk 1]
```diff
@@ -262,28 +252,20 @@ public function getContributorType( $row ) {
...
+	public function saveVotes() {
 		$pageIds = array_keys( $this->getPages() );
 
-		$dumper = $this;
 		$dbh = wfGetDB( DB_SLAVE );
 		( new \WikiaSQL() )
 			->SELECT_ALL()
 			->FROM( self::TABLE_VOTE )
 			->WHERE( 'article_id' )->IN( $pageIds )
 			->runLoop( $dbh, function ( &$pages, $row ) {
```
Comment (WikiaTech) [inline extensions/wikia/Discussions/maintenance/ForumDumper.php:263]: MAJOR Remove the unused function parameter "$pages". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)

File: extensions/wikia/Discussions/maintenance/dumpForumData.php [hunk 2]
```diff
@@ -45,40 +47,19 @@ public function dumpPages() {
 		$pages = $this->dumper->getPages();
 
 		foreach ( $pages as $id => $data ) {
```
Comment (WikiaTech) [inline extensions/wikia/Discussions/maintenance/dumpForumData.php:49]: MAJOR Remove this unused "$id" local variable. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1481)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 13 issues
* MAJOR 6 major
* MINOR 7 minor

Watch the comments in this conversation to review them.

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. MAJOR [ForumDumper.php#L116](https://github.com/Wikia/app/blob/76b02c72527a62b011ab600910d671f8b6b1e34a/extensions/wikia/Discussions/maintenance/ForumDumper.php#L116): Remove the unused function parameter "$pages". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. MAJOR [ForumDumper.php#L153](https://github.com/Wikia/app/blob/76b02c72527a62b011ab600910d671f8b6b1e34a/extensions/wikia/Discussions/maintenance/ForumDumper.php#L153): Remove the unused function parameter "$revisions". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. MAJOR [ForumDumper.php#L239](https://github.com/Wikia/app/blob/76b02c72527a62b011ab600910d671f8b6b1e34a/extensions/wikia/Discussions/maintenance/ForumDumper.php#L239): Reduce the number of returns of this function 4, down to the maximum allowed 3. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1142)
1. MAJOR [dumpForumData.php#L23](https://github.com/Wikia/app/blob/76b02c72527a62b011ab600910d671f8b6b1e34a/extensions/wikia/Discussions/maintenance/dumpForumData.php#L23): Remove this unused "$required" local variable. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1481)
1. MINOR [ForumDumper.php#L33](https://github.com/Wikia/app/blob/76b02c72527a62b011ab600910d671f8b6b1e34a/extensions/wikia/Discussions/maintenance/ForumDumper.php#L33): Define a constant instead of duplicating this literal "page_id" 7 times. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)
1. MINOR [ForumDumper.php#L34](https://github.com/Wikia/app/blob/76b02c72527a62b011ab600910d671f8b6b1e34a/extensions/wikia/Discussions/maintenance/ForumDumper.php#L34): Define a constant instead of duplicating this literal "namespace" 3 times. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)
1. MINOR [ForumDumper.php#L40](https://github.com/Wikia/app/blob/76b02c72527a62b011ab600910d671f8b6b1e34a/extensions/wikia/Discussions/maintenance/ForumDumper.php#L40): Define a constant instead of duplicating this literal "length" 4 times. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)
1. MINOR [ForumDumper.php#L58](https://github.com/Wikia/app/blob/76b02c72527a62b011ab600910d671f8b6b1e34a/extensions/wikia/Discussions/maintenance/ForumDumper.php#L58): Define a constant instead of duplicating this literal "page_namespace" 3 times. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)
1. MINOR [ForumDumper.php#L61](https://github.com/Wikia/app/blob/76b02c72527a62b011ab600910d671f8b6b1e34a/extensions/wikia/Discussions/maintenance/ForumDumper.php#L61): Define a constant instead of duplicating this literal "user_identifier" 4 times. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)
1. MINOR [ForumDumper.php#L62](https://github.com/Wikia/app/blob/76b02c72527a62b011ab600910d671f8b6b1e34a/extensions/wikia/Discussions/maintenance/ForumDumper.php#L62): Define a constant instead of duplicating this literal "timestamp" 4 times. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)

**Comment by slayful** [comment]
Obsolete.

**Pull request closed without merging**